cfg/rippled-example.cfg

```diff
@@ -1158,7 +1158,7 @@ protocol = ws
 #-------------------------------------------------------------------------------
 
 [node_size]
-medium
+huge
```

Comment:
I don't think that **`tiny`** or **`huge`** are sensible defaults but, between the two, **`tiny`** is the safer option and may be more appropriate. Ideally, we would try to auto-detect the appropriate value for this instead of just picking a "default" value in the config.

At a minimum we need to pick a value and stick with it. As @MarkusTeufelberger pointed out, we claim the default is "tiny" earlier in the config and, true enough, it is the default if nothing else is specified but something _is_ specified here.

Reply:
That makes sense. I will close this PR.